changelogs/fragments/647-removal-hint.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - "When announcing collection removals and upcoming removals in the changelog, add a note that that can still be installed manually
```

Comment:
```suggestion
  - "When announcing collection removals and upcoming removals in the changelog, add a note that users can still manually install removed collections
```

Reply:
I didn't suggest it but was thinking about "pending" over "upcoming". Then I looked it up and found this explanation between those two words:

"As adjectives the difference between pending and upcoming is that pending is awaiting a conclusion or a confirmation while upcoming is happening or appearing in the relatively near future."

I'd say "upcoming" is the right choice here but wanted to point this info out all the same. Cheers.